testsV2/ut_repyv2api_openfileconsumesfilehandles.py

```diff
@@ -19,12 +19,20 @@
 
 try:
   fileh = openfile(TRY_FILE, True)
+  fileh.close()
 except ResourceExhaustedError:
   pass
 else:
   log("Opened file with no handles available!",'\n')
-
 # Check if the file exists
 if TRY_FILE in listfiles():
   log("The file was created with no handles available!",'\n')
+  removefile(TRY_FILE)
+
+#Close all random files opened
+for randFile in filehandles:
+  randFile.close()
```

Comment:
General variable name should always user lower case. https://seattle.poly.edu/wiki/CodingStyle#NamingConventions